docker/Dockerfile.release

```diff
@@ -1,66 +0,0 @@
-#
-# Copyright 2016-2020 The OpenZipkin Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
-# in compliance with the License. You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under
-# the License.
-#
-
-# Download artifact from bintray, where the artifact should have been published by now.
-
-FROM openzipkin/zipkin-builder as built
-
-ARG RELEASE_VERSION
-RUN test -n "$RELEASE_VERSION"
-
-# source_branch is something like
-#
-#   master       - Building the master branch. This cut command will return 'master', and Dockerfiles will ignore it
-#   1.0.1        - Building a release image along with a new Zipkin server version. This cut command will return
-#                  '1.0.1' and Dockerfiles will use it to fetch the correct version of Zipkin.
-#   docker-1.0.1 - Building a release image, but not a new Zipkin server. This cut command will return
-#                  '1.0.1' and Dockerfiles will use it to fetch the correct version of Zipkin.
-
-WORKDIR /zipkin-dependencies
-
-# Download jars using Maven. It will try to resolve the artifact from Maven Central, which might
-# not work right away if the sync is taking time, followed by bintray, which should always work
-# since it's where we publish to.
-RUN mvn org.apache.maven.plugins:maven-dependency-plugin:get \
-    -DremoteRepositories=bintray::::https://dl.bintray.com/openzipkin/maven -Dtransitive=false \
-    -Dartifact=io.zipkin.dependencies:zipkin-dependencies:${RELEASE_VERSION}:jar && \
-  cp ~/.m2/repository/io/zipkin/dependencies/zipkin-dependencies/${RELEASE_VERSION}/zipkin-dependencies-${RELEASE_VERSION}.jar .
-
-FROM openzipkin/jre-full:1.8.0_252-b09
```

Comment:
`# Until we can upgrade to Spark 3, we are pinned to JRE 1.8` comment was also not copy/pastad